mods/persistence/controllers/subsystems/initialization/chargen.dm

```diff
@@ -28,16 +27,99 @@ SUBSYSTEM_DEF(chargen)
 				CHECK_TICK
 
 /datum/controller/subsystem/chargen/proc/get_spawn_turf()
-	return pick(chargen_spawns)
+	//Grab first free pod
+	for(var/area/chargen/A in chargen_areas)
+		if(chargen_areas[A] > 0)
+			continue
+		var/obj/abstract/landmark/chargen_spawn/L = locate(/obj/abstract/landmark/chargen_spawn) in A
+		var/turf/spawnturf = L? get_turf(L) : null
+		if(!L)
+			log_warning("Area '[log_info_line(A)]' contains no '/obj/abstract/landmark/chargen_spawn' for spawning characters..")
```

Comment:
Extra full stop at the end.

Reply:
That was meant to be an ellipsis actually...